### PR TITLE
Cleanup defaults

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
  * [ ] Bug fix
  * [ ] Feature addition
  * [x] Refactoring
  * [ ] Minor / simple change (like a typo)
  * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): PICARD-XXX

# Solution

# Action

Additional actions required:
* [ ] Update Picard [documentation](https://github.com/metabrainz/picard-docs) (please include a reference to this PR)
* [ ] Other (please specify below)

### CODE REVIEW

**Comment by zas** [comment]
@phw can you test on macos (in case of)?

Also shouldn't [`DEFAULT_STARTING_DIR`](https://github.com/metabrainz/picard/blob/4af62cc88baccebcdc5627d2663717c10b31baec/picard/const/defaults.py#L127) be the same as [`DEFAULT_CURRENT_BROWSER_PATH`](https://github.com/metabrainz/picard/blob/4af62cc88baccebcdc5627d2663717c10b31baec/picard/const/defaults.py#L80-L101) ? Current code makes them different on MacOs.

File: picard/util/macos.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    try:
+        for entry in os.scandir("/Volumes/"):
+            if entry.is_symlink() and os.path.realpath(entry.path) == "/":
+                return entry.path
+    except OSError:
+        log.warning("Could not detect macOS boot volume", exc_info=True)
+    return None
+
+
+def extend_root_volume_path(path):
```
Comment (phw) [inline picard/util/macos.py:36]: We could have basic tests for this function, which run only on macOS. We cannot test for the exact volume path, but:

- Any path passed to the function starting with `/Volumes/` must be returned unchanged
- In any other case the resulting path must:
  - start with `/Volumes/` 
  - end with the original `path`

Comment (zas) [reply picard/util/macos.py:36]: Tests added in https://github.com/metabrainz/picard/pull/2438/commits/fbab1f1884e6d39f687b4c1055a66e9299555b34

**Review by phw — APPROVED** [review]
Tested on macOS, works as before. So let's merge it.

The handling of the default paths should indeed be changed a bit, I'll open a separate PR for this.

**Pull request merged**